01_fib/fib.go

```diff
@@ -0,0 +1,21 @@
+package main
```

Comment:
Tests are missing.
Please consider adding corner cases (n: -1, 0, and a very large number)